Makefile.in

```diff
@@ -42,7 +42,7 @@ DISTFILES=$(SRC) \
 all:	mod_auth_mellon.la
 
 mod_auth_mellon.la: $(SRC) auth_mellon.h auth_mellon_compat.h
-	@APXS2@ -Wc,"-std=c99 @MELLON_CFLAGS@ @OPENSSL_CFLAGS@ @LASSO_CFLAGS@ @CURL_CFLAGS@ @GLIB_CFLAGS@ @CFLAGS@" -Wl,"@OPENSSL_LIBS@ @LASSO_LIBS@ @CURL_LIBS@ @GLIB_LIBS@" -Wc,-Wall -Wc,-g -c $(SRC)
+	@APXS2@ -Wc,"-std=c99 @MELLON_CFLAGS@ @OPENSSL_CFLAGS@ @LASSO_CFLAGS@ @CURL_CFLAGS@ @GLIB_CFLAGS@ @CFLAGS@ @LIBXML2_CFLAGS@ @XMLSEC_CFLAGS@ @CFLAGS@" -Wl,"@OPENSSL_LIBS@ @LASSO_LIBS@ @CURL_LIBS@ @GLIB_LIBS@ @LIBXML2_LIBS@ @XMLSEC_LIBS@" -Wc,-Wall -Wc,-g -c $(SRC)
```

Comment:
This is a nit, but is it necessary that CFLAGS is added again? (Is there maybe some ordering issue?)

Reply:
Unfortunately I don't know anything about `automake` and friends, so I'll trust that you're right on that and it was a mistake (this patch exists in my OS). Would you like the first or the second removed?

Reply:
I think the second is extra, the one after `XMLSEC_CFLAGS`. Do you have a way to test if that patch still would work on your OS?

Reply:
I do. I'll push here, test again, and report :+1: Thanks for your guidance.

Reply:
This new commit compiles fine for me. Looks like we're good to go.

Reply:
And CI is also green. Thank you for the PR!